docker-for-windows/index.md

```diff
@@ -493,23 +367,21 @@ You can set up your own [registries](/registry/introduction.md) on the **Basic**
 Daemon settings.
 
 As an alternative to using [Docker Hub](https://hub.docker.com/) to store your
-public or private images or [Docker Trusted
-Registry](/datacenter/dtr/2.1/guides/index.md), you can use Docker to set up
-your own insecure [registry](/registry/introduction.md). Add URLs for insecure
-registries and registry mirrors on which to host your images. (See also, [How do
-I add custom CA certificates?](faqs.md#how-do-i-add-custom-ca-certificates) and [How do I add client certificates?](faqs.md#how-do-i-add-client-certificates) in
-the FAQs.)
+public or private images or [Docker Trusted Registry](/datacenter/dtr/2.1/guides/index.md),
```

Comment:
Is it ok to pin a version number?  Shouldn't we rather point to the latest doc?

Reply:
I agree ... but that's how those docs are being designed. It causes problems with linking, I know.